### ISSUE TITLE
Automatically added `from` parameter limits results when changing criteria

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/wpt.fyi#1506]
Steps to repo:
- Go to https://wpt.fyi/runs, be redirected to https://wpt.fyi/runs?label=master&from=2019-09-16T00%3A00
- Click "edit" and enter "beta" in the labels field. That'll request the beta channel for all products.
- Submit and end up at https://wpt.fyi/runs?label=master&label=beta&from=2019-09-15T22%3A00

This is all that is shown:
![image](https://user-images.githubusercontent.com/498917/65415006-23459700-ddf5-11e9-8af1-f88ea1b05b3c.png)

Removing `from=2019-09-15T22%3A00` from the URL shows many more runs, going back to Oct 1, 2018, which I think is when we introduced beta runs.

In other words, the `from` parameter is tainting results.

It would be nice if this just never shows up in the URL unless a user explicitly requests a time range in the from/to fields.
